Packages/tlp.udonvoiceutils/Runtime/Examples/VoiceOverrideTriggerZone.cs

```diff
@@ -23,13 +23,17 @@ public class VoiceOverrideTriggerZone : TlpBaseBehaviour
 
         private Collider[] _allTrigger;
 
+        private int[] playerIds = new int[82];
+        private int[] colliderStacks = new int[82];
+
         public override void Start() {
             base.Start();
             // ensure that players already being inside the trigger before udon starts are detected by disabling them
             // once and enabling them 1 frame later again  
             _allTrigger = gameObject.GetComponents<Collider>();
             DisableAllTrigger();
             SendCustomEventDelayedFrames(nameof(EnableAllTriggerDelayed), 1, EventTiming.LateUpdate);
+            SendCustomEventDelayedSeconds(nameof(TriggerStayCheck), 1f, EventTiming.LateUpdate);
```

Comment:
Depending on the scene setup this can have a major impact on performance when many zones update all at once.

Additionally, is guaranteed that this custom loop keeps running should this script be disabled?